repose-aggregator/artifacts/cli-utils/src/main/java/org/openrepose/cli/command/datastore/local/ReposeJMXClient.java

```diff
@@ -36,6 +36,9 @@
 // close jmxc, but how on Earth would we test it since cli-utils doesn't seem to be used?  Suppressing this for now.
 public class ReposeJMXClient implements ReposeLocalCacheMBean {
 
+    private static final String hostname = NetUtilities.bestGuessHostname();
+    private static final String REPOSE_LOCAL_CACHE_OBJECT_NAME = hostname + ":001=\"org\",002=\"openrepose\",003=\"core\",004=\"services\",005=\"datastore\",006=\"distributed\",007=\"impl\",008=\"ehcache\",009=\"ReposeLocalCache\"";
+
```

Comment:
Super gross, I know, but semi-necessary because the hostname portion of the object name is dynamic.

Yes, we could do a query for the bean instead, but when I looked down that route, things got messier.